### PR TITLE
Added description for in the ShapeList class of mensuration file.

### PR DESCRIPTION
**Pull request by vk4s**
..... Blah Blah Blah ...

### CODE REVIEW

**Comment by vk4s** [comment]
Hey merge this PR so that I can contribute in the project.

File: mini_projects/Math_utils/mensuration.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 
 class ShapeList:
+    '''Defining constants for variousshape'''
```
Comment (ak4shp) [inline mini_projects/Math_utils/mensuration.py]: There is typo.

Comment (vk4s) [reply mini_projects/Math_utils/mensuration.py]: Ok I will fix that.

File: mini_projects/Math_utils/mensuration.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 
 class ShapeList:
-    '''Defining constants for variousshape'''
+    '''Defining constants for various shapes'''
```
Comment (ak4shp) [inline mini_projects/Math_utils/mensuration.py:21]: looks good.

Comment (vk4s) [reply mini_projects/Math_utils/mensuration.py:21]: thanks

**Pull request merged**